### PR TITLE
Hexagon build src tests

### PR DESCRIPTION
**Pull request by 0001vrn**

### CODE REVIEW

File: buildSrc/src/test/kotlin/FileRangeTest.kt [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import org.testng.annotations.BeforeMethod
+import org.testng.annotations.Test
+import java.io.File
+
+class FileRangeTest {
```
Comment (jaguililla) [inline buildSrc/src/test/kotlin/FileRangeTest.kt]: I would also add some corner cases like:

* Not present tag
* Not ended tag
* Empty text range

Also, It would be great to include more specific asserts like asserting the expected text, or number of lines.

Comment (0001vrn) [reply buildSrc/src/test/kotlin/FileRangeTest.kt]: Resolved first two corner cases. And fixed asserts as suggested.

Comment (jaguililla) [reply buildSrc/src/test/kotlin/FileRangeTest.kt]: Great! I just gave you more details about the other test scenario at Slack.

I also wanted to note that you need to do the PR to the `develop` branch. Making it to `master` would publish a new release and fail because of the version number. Sorry for the inconvenience!

Comment (0001vrn) [reply buildSrc/src/test/kotlin/FileRangeTest.kt]: Okay! I have understood the scenario. Expect a new PR(check out from `develop`) soon.

**Comment by 0001vrn** [comment]
Will checkout `develop`. Changes will be merged from that branch.

**Comment by jaguililla** [comment]
Just for the record, you can do that without closing and making a different PR :)

You can do `git pull origin develop` to your branch and push the changes to your fork's branch (this will update the PR).

And if you edit the PR, you can make it point to a different branch.
